libpod/container_path_resolution.go

```diff
@@ -128,7 +128,7 @@ func isPathOnVolume(c *Container, containerPath string) bool {
 		if cleanedContainerPath == filepath.Clean(vol.Dest) {
 			return true
 		}
-		for dest := vol.Dest; dest != "/"; dest = filepath.Dir(dest) {
+		for dest := vol.Dest; dest != "/" && dest != "."; dest = filepath.Dir(dest) {
```

Comment:
IDK, but wonder if it would be better to add at line 134.5

```
if dest == "." {
    return false}
}
```

Reply:
To me it looks good next to the check for `"/"`.